### PR TITLE
Add xrp

### LINKED ISSUE #91: [ADD] XRP support
Please add XRp support to registry and FE

### PR DESCRIPTION
**Pull request by Ehsan-saradar**
close #91

### CODE REVIEW

File: internal/address/xrp.go [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+	sha.Write(hash)
+	hash = sha.Sum(nil)
+
+	checksum := hash[:4]
+
+	finalHash := append(versionHash, checksum...)
+	return base58Encode(finalHash), nil
+}
+
+func base58Encode(input []byte) string {
```
Comment (johnnyluo) [inline internal/address/xrp.go:46]: there is a method in `"github.com/cosmos/btcutil/base58"`, so we don't need to create our own

File: internal/address/xrp.go [hunk 2]
```diff
@@ -0,0 +1,73 @@
...
+	hash = sha.Sum(nil)
+
+	sha = sha256.New()
+	sha.Write(hash)
+	hash = sha.Sum(nil)
+
+	checksum := hash[:4]
+
+	finalHash := append(versionHash, checksum...)
+	return base58Encode(finalHash), nil
```
Comment (johnnyluo) [inline internal/address/xrp.go:43]: Please use `base58.Encode`

**Review by johnnyluo — CHANGES_REQUESTED** [review]
see comments

File: internal/address/xrp.go (hunk 1 above)
Comment (Ehsan-saradar) [reply internal/address/xrp.go]: Fixed

File: internal/address/xrp.go (hunk 2 above)
Comment (Ehsan-saradar) [reply internal/address/xrp.go]: Resolved

**Review by johnnyluo — APPROVED** [review]
LGTM

**Pull request merged**